src/main/java/codechicken/lib/internal/ClientInit.java

```diff
@@ -74,4 +78,9 @@ private static void onRegisterGeometryLoaders(ModelEvent.RegisterGeometryLoaders
         event.register("item_composite", new CompositeItemModel());
         event.register("class", new ClassModelLoader());
     }
+
+    public static void onResourceReload(RegisterClientReloadListenersEvent event) {
+        event.registerReloadListener(CCGuiTextures.getAtlasHolder());
+        CursorHelper.onResourceReload();
```

Comment:
Why is this not on a reload listener? Why do we just call it? Does this event get called multiple times?